docs/spring-on-liberty/README.md

```diff
@@ -0,0 +1,55 @@
+# Passing configuration at runtime to SprinBoot WAR running on WebSphere Liberty
+
+### What do we need ?
+
+The SpringBoot WAR file is deployed on WebSphere Liberty Profile. When it is deployed to the server, the application should be access to get the configuration values at run time. There is no access to the code and all we can access is the application `war` file. Also, the application is running on Liberty CLI and `./server run` should do everything.
+
+### Our Approach
+
+We are using [WebSphere Liberty WebProfile 8](https://developer.ibm.com/wasdev/downloads/download-latest-stable-websphere-liberty-runtime/) to run the demo application. For this usecase, we are using [Inventory service](https://github.com/ibm-cloud-architecture/refarch-cloudnative-micro-inventory/tree/spring) which is a part of our [BlueCompute](https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes) application.
+
+Initially, we packaged the SpringBoot application as a `war` file. Let us now deploy this `war` file on WebSphere Liberty. You can do this by placing the file in `dropins` folder of the server. You can place it here `wlp > usr > servers > defaultServer > dropins > Spring > YourApplication.war`.
+
+<p align="center">
+    <img src="images/springbootwar.png" height=500 width=500>
+</p>
+
+Run `./server run` to start your application. When you do this, your application will pick up the configuration values from the below yaml which is a part of the application and this is packaged along with the application in the `war` file.
+
+<p align="center">
+    <img src="images/configvalues.png" height=400 width=500>
+</p>
+
+If you want to pass the configuration values other than the ones specified in the above file, you need to override these values by passing them externally. You can do this by placing the values in `jvm.options`. You can create a file names `jvm.option` under the `shared` folder that comes with the server as below.
```

Comment:
Typo in the 3rd sentence - "create a file `named`..."

Reply:
Corrected